scripts/update-dependencies/Program.cs

```diff
@@ -102,7 +103,7 @@ private static IEnumerable<IDependencyUpdater> GetUpdaters()
             yield return CreateRegexPropertyUpdater(config.DependencyFilePath, "SystemNumericsVectorsVersion", "System.Numerics.Vectors");
             yield return CreateRegexPropertyUpdater(config.DependencyFilePath, "SystemBuffersVersion", "System.Buffers");
             yield return CreateRegexPropertyUpdater(config.DependencyFilePath, "SystemIOPipelinesVersion", "System.IO.Pipelines");
-            yield return CreateFileUpdater(config.CLIVersionFilePath, "Microsoft.DotNet.Cli.Utils");
+            yield return CreateFileUpdater(config.CLIVersionFilePath, "core-sdk");
```

Comment:
Should we rename this (and the file name) to something more appropriate now (something related to CoreSDK?
https://github.com/dotnet/corefxlab/blob/511aa547cf87eda83e2fc86f5e86132d73c44b18/scripts/update-dependencies/Config.cs#L37

Reply:
Probably. However, after moving to Arcade, this file won't exist anymore, so it might not be worth the effort. (It's also still called DotnetCLIVersion.txt in CoreFX master, for what it's worth.)

Reply:
Sounds good to me.